Source/common/SNTConfigurator.m

```diff
@@ -1242,13 +1245,36 @@ - (NSMutableDictionary *)readForcedConfig {
          ![obj isKindOfClass:[NSString class]])) {
       continue;
     }
+
     forcedConfig[key] = obj;
+    overridesApplied = YES;
+
     if (self.forcedConfigKeyTypes[key] == [NSRegularExpression class]) {
       NSString *pattern = [obj isKindOfClass:[NSString class]] ? obj : nil;
       forcedConfig[key] = [self expressionForPattern:pattern];
     }
   }
+
+  if (overridesApplied) {
+    NSLog(@"WARNING: Running with overrides applied!");
```

Comment:
Should this be an asl_log like we do in SNTLogging?

Reply:
Since we unfortunately can't include SNTLogging within the configurator as things are currently setup, and since this print is only for local debug builds and doesn't meed to conform to any structure, I went with the simple route here. Was there any sort of structure/data you were hoping for as is currently emitted by something like `LOGW`?

Reply:
Yeah pretty much was looking to keep things uniform and include the warning flag, also wanted to make sure the logs turned up in the same place for debugging purposes.

Reply:
As discussed, opting to just remove the log instead of recreating all of that logic just for debug builds.